package-lock.json

```diff
@@ -6305,7 +6305,7 @@
 		},
 		"ncp": {
 			"version": "2.0.0",
-			"resolved": "https://registry.npmjs.org/ncp/-/ncp-2.0.0.tgz",
+			"resolved": "http://registry.npmjs.org/ncp/-/ncp-2.0.0.tgz",
```

Comment:
please keep https here, to fix this locally, remove node_modules folder, sorrect it here and run npm i again

Reply:
reverted